GameServerLib/Program.cs

```diff
@@ -10,9 +10,9 @@ namespace LeagueSandbox.GameServer
     /// </summary>
     public class GameServerLauncher
     {
-        public static void LaunchServer(string configPath, ushort serverPort)
+        public static void LaunchServer(string configPath, ushort serverPort, string configJSON)
```

Comment:
Let's only accept a ready `Config` object here like explained in another comment